spec/policies/defence_request_controller_policy_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe DefenceRequestControllerPolicy do
+  subject { DefenceRequestControllerPolicy }
```

Comment:
You could right this as `subject { described_class }`. Also you could add a custom example group which sets this by default if there are going to be multiple policies.